util/automation.py

```diff
@@ -32,8 +32,12 @@
 
 DEEPSPEECH_CLONE_PATH  = os.path.abspath(os.environ.get('ds_clone_path', './ds_exec_clone/'))
 
-CACHE_DIR = os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer')
-DATA_DIR  = os.path.join(BaseDirectory.xdg_data_home, 'deepspeech_wer')
+# Cache and data should be used for small amounts of data
+CACHE_DIR     = os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer')
+DATA_DIR      = os.path.join(BaseDirectory.xdg_data_home, 'deepspeech_wer')
```

Comment:
It looks like DATA_DIR is no longer used, why not just change this instead of introducing HUGE_DATA_DIR? Also, shouldn't this follow ${dataroot} from run-wer-automation?